prover/README.md

```diff
@@ -0,0 +1,30 @@
+## How-to run Snarkify prover locally
+1. `make snarkify`
```

Comment:
what does it mean? install the snarkify cli?

Reply:
no this is a build target in the Makefile to build the snarkify binary from Rust code